### PR TITLE
portability for sumatra

### PR DESCRIPTION
**Pull request by babsey**
This PR allows user to move Sumatra project folder to other path when this folder contains all its compartments (scripts, parameter files and data). 

I once tried to pull this PR. I hope this time you can test it and give me feedback of your opinion.

### CODE REVIEW

**Comment by apdavison** [comment]
Please could you fix the failing test?

**Comment by babsey** [comment]
I failed to solve the problem why sumatra creates old project folder after moving the project to other path. I assume that it occurs FileSystemDataStore.root in datastore/filesystem.py

When I changed it from (line 76)
`self.root = os.path.abspath(root or './Data')`
to
```
if os.path.exists(root):
    self.root = os.path.abspath(root)
else:
    self.root = os.path.abspath('./Data')
```
some tests failed.

File: sumatra/commands.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ def init(argv):
                             description=description)
     parser.add_argument('project_name', metavar='NAME', help="a short name for the project; should not contain spaces.")
     parser.add_argument('-d', '--datapath', metavar='PATH', default='./Data', help="set the path to the directory in which smt will search for output datafiles generated by the simulation/analysis. Defaults to %(default)s.")
-    parser.add_argument('-i', '--input', metavar='PATH', default='/', help="set the path to the directory relative to which input datafile paths will be given. Defaults to the filesystem root.")
+    parser.add_argument('-i', '--input', metavar='PATH', default='.', help="set the path to the directory relative to which input datafile paths will be given. Defaults to the filesystem root.")
```
Comment (apdavison) [inline sumatra/commands.py]: Is this change necessary for portability? 

Having the global root as the default root for the input data store avoids problems in the common case that input data are stored separately from code.

Comment (babsey) [inline sumatra/commands.py]: I had also thought about this and I like the idea that the Sumatra project folder should be complete enclosed. If you want to implement input data, you can create a symbolic link to the Sumatra project folder.

Furthermore, for the project file I use relative path for datastores. It would look ugly like '../../../../', if we use '/' as default root for input data store. After moving the folder to other path, we have to change the default root for input data store.

@other: What do you think?

**Comment by babsey** [comment]
I close this PR because it is stale.
